.spelling

```diff
@@ -234,6 +238,7 @@ FQDNs
 FQDN
 frontend
 gbd
+GB
```

Comment:
Remove this. Replace GB in your doc with  GB surrounded by backticks similar to other pages (See Docker Desktop for example).

Reply:
this is better than my last suggestion.